bin/update-version.sh

```diff
@@ -38,7 +38,7 @@ sed_inplace "s/Stable tag: .*  $/Stable tag: $VERSION  /" README.md
 sed_inplace "s/\"version\": \".*\"/\"version\": \"$VERSION\"/" package.json
 sed_inplace "s/export const PLUGIN_VERSION = '.*'/export const PLUGIN_VERSION = '$VERSION'/" tests/e2e/utils.ts
 sed_inplace "s/ \* Version:           .*$/ \* Version:           $VERSION/" wp-parsely.php
-sed_inplace "s/const PARSELY_VERSION = '.*'/const PARSELY_VERSION = '$VERSION'/" wp-parsely.php
+sed_inplace "s/const PARSELY_VERSION             = '.*'/const PARSELY_VERSION             = '$VERSION'/" wp-parsely.php
```

Comment:
How about this, so if we add another long constant it doesn't break?

```suggestion
sed_inplace "s/const PARSELY_VERSION\( *\)= '.*'/const PARSELY_VERSION\1= '$VERSION'/" wp-parsely.php
```

This takes any number of spaces `( *)` and reuses it in reference `\1`. Try it locally with different spaces, it should work!

We could also do this for `Version:` in the plugin header, but that alignment is less likely to change.

Reply:
CodeRabbit had suggested something similar in its "Nitpick comments" but I actually prefer to be more rigid in this case, in order to avoid any possible additional changes (although I admit the possibility is very very low).

Let me know if this is OK with you.